tests/cephfs/cephfs_mirroring/cephfs_mirroring_utils.py

```diff
@@ -451,31 +467,30 @@ def validate_snapshot_sync_status(
             json.JSONDecodeError: If there's an error decoding JSON from the Ceph mirror status response.
         """
         log.info("Get peer mirror status")
-        try:
-            log.info("Validate the snapshot sync to target cluster.")
-            cmd = (
-                f"cd /var/run/ceph/{fsid}/ ; ceph --admin-daemon {asok_file} fs mirror peer status "
-                f"{fs_name}@{filesystem_id} {peer_uuid} -f json"
-            )
-            out, _ = cephfs_mirror_node.exec_command(sudo=True, cmd=cmd)
-            data = json.loads(out)
-            for path, status in data.items():
-                last_synced_snap = status.get("last_synced_snap")
-                if last_synced_snap:
-                    if last_synced_snap.get("name") == snapshot_name:
-                        sync_duration = last_synced_snap.get("sync_duration")
-                        sync_time_stamp = last_synced_snap.get("sync_time_stamp")
-                        snaps_synced = status.get("snaps_synced")
-                        return {
-                            "snapshot_name": snapshot_name,
-                            "sync_duration": sync_duration,
-                            "sync_time_stamp": sync_time_stamp,
-                            "snaps_synced": snaps_synced,
-                        }
-            return None
-        except json.JSONDecodeError as e:
-            log.error(f"JSON decoding error: {e}")
-            return None
+        log.info("Validate the snapshot sync to target cluster.")
+        cmd = (
+            f"cd /var/run/ceph/{fsid}/ ; ceph --admin-daemon {asok_file} fs mirror peer status "
+            f"{fs_name}@{filesystem_id} {peer_uuid} -f json"
+        )
+        out, _ = cephfs_mirror_node.exec_command(sudo=True, cmd=cmd)
+        data = json.loads(out)
+        for path, status in data.items():
+            last_synced_snap = status.get("last_synced_snap")
+            if last_synced_snap:
+                if last_synced_snap.get("name") == snapshot_name:
+                    sync_duration = last_synced_snap.get("sync_duration")
+                    sync_time_stamp = last_synced_snap.get("sync_time_stamp")
+                    snaps_synced = status.get("snaps_synced")
+                    return {
+                        "snapshot_name": snapshot_name,
+                        "sync_duration": sync_duration,
+                        "sync_time_stamp": sync_time_stamp,
+                        "snaps_synced": snaps_synced,
+                    }
+                    log.info("ALl snapshots are synced")
+        else:
+            log.error(f"{snapshot_name} not synced")
```

Comment:
Can you add last sync details also in the error log 
log.error(f"{snapshot_name} not synced") --> log.error(f"{snapshot_name} not synced, \n last synced data is {data}")